packages/mesos/build

```diff
@@ -21,6 +21,7 @@ LDFLAGS="-Wl,-rpath,/opt/mesosphere/lib" "/pkg/src/mesos/configure" \
   --enable-libevent --enable-ssl \
   --enable-install-module-dependencies \
   --enable-grpc \
+  --enable-jemalloc-allocator \
```

Comment:
I thought about it, but decided against it on the grounds that it would be a massiva PITA with no real benefit until we have a second application also using jemalloc as memory allocator.

Reply:
Please call this out in the commit message. Somebody might come along in the future wanting to use jemalloc and wonder if there is any reason Mesos shouldn't use an unbundled one.

Reply:
Done.